src/resources/js/test/usersTab.test.tsx

```diff
@@ -255,7 +255,7 @@ describe('check for correct lists of users in User dropdown menu', () => {
     const user = await startWithUsersTab(mockServerAdmin);
     // The default should be 'Create New User'.  This default should be what is shown before
     // clicking on the dropdown
-    await screen.findByText(/Create New User/);
+    await screen.findByRole('button', { name: /user create new user/i });
```

Comment:
change to 
await screen.findByRole('button', { name: /create new user/i });

Reply:
good catch